pkgs/applications/networking/cluster/docker-machine-kvm/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, buildGoPackage, fetchFromGitHub, libvirt, libvirt-glib, kvm, docker-machine }:
+
+buildGoPackage rec {
+    name = "docker-machine-kvm-${version}";
+    version = "20160801-${stdenv.lib.strings.substring 0 7 rev}";
```

Comment:
we usually prefer released versions instead of nightly builds.
To achieve with golang just go to project directory and say:

```
git checkout v0.7.0
```